src/cap32.h

```diff
@@ -266,22 +259,22 @@ typedef struct {
    unsigned int kbd_layout;
 
    unsigned int max_tracksize;
-   std::string snap_path;
-   std::string snap_file;
-   bool snap_zip;
+
+   std::string snap_path; // Path where images will be loaded/saved by default. Only one exposed in CFG file
```

Comment:
Replace 'images' by 'snapshot' in comments ? (not necessarly less confusing ... how to ensure there is no confusion with screenshot ?)